addon/components/bs4/bs-form/element/label.js

```diff
@@ -1,9 +1,65 @@
+import Ember from 'ember';
 import FormElementLabel from 'ember-bootstrap/components/base/bs-form/element/label';
 
+const {
+  computed
+} = Ember;
+
 export default FormElementLabel.extend({
   tagName: 'label',
 
-  classNames: ['form-control-label'],
-  classNameBindings: ['invisibleLabel:sr-only', 'labelClass'],
-  attributeBindings: ['formElementId:for']
+  classNames: [],
+  classNameBindings: ['invisibleLabel:sr-only', 'isHorizontal:col-form-label', 'isCheckbox:form-check-label', 'labelClass'],
```

Comment:
My bad, you're right [#260](https://github.com/kaliber5/ember-bootstrap/pull/260) does overlap. The only difference is the use of the `isHorizontal` check. 

I have seen the same issue that is mentioned, when i was looking at the BS4 documentation. I think the use of `form-control-label` is a (copy from BS3?) mistake in the BS4 documentation. At least 90% of the BS4 form documentation uses `form-control-label`, i think it is ok to use it.